app/lint.xml

```diff
@@ -1,8 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <lint>
-    <issue id="NewerVersionAvailable" severity="ignore">
-        <!-- We're using dependabot to keep dependencies up-to-date -->
-    </issue>
+    <!-- We're using dependabot to keep dependencies up-to-date -->
+    <issue id="NewerVersionAvailable" severity="ignore"/>
+    <issue id="GradleDependency" severity="ignore"/>
```

Comment:
Ported to #63 to fix the latest dependabot PRs which are currently broken due to this.